sdk/nestjs/package.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "@devcycle/nestjs-server-sdk",
+    "version": "0.0.1",
+    "description": "The DevCycle NestJS Server SDK used for feature management.",
+    "license": "MIT",
+    "dependencies": {
+        "@devcycle/nodejs-server-sdk": "^1.18.3",
+        "@devcycle/types": "^1.9.0",
+        "@nestjs/common": "^10.3.3",
```

Comment:
these need to be peer dependencies

Reply:
yeah i was talking about the nest ones specifically

Reply:
ah, unless you were referring to "these" as the nestjs packages below this one, rather than the others shown in this preview 👀